modules/installation-overview.adoc

```diff
@@ -77,15 +65,25 @@ of new certificates or keys, after installation.
 == Available platforms
 
 In {product-title} {product-version}, you can install a cluster that uses
-installer-provisioned infrastructure on only Amazon Web Services (AWS).
+installer-provisioned infrastructure on the following platforms:
+
+* Amazon Web Services (AWS)
+* Google Cloud Platform (GCP)
+* Microsoft Azure
+* OpenStack
+
+For these clusters, all machines, including the computer that you run the installation process on, must have direct internet access to pull images for platform containers and provide telemetry data to Red Hat.
 
 In {product-title} {product-version}, you can install a cluster that uses
 user-provisioned infrastructure on the following platforms:
 
 * AWS
+* GCP
 * VMware vSphere
 * Bare metal
 
+With installations on user-provisioned infrastructure, each machine can have full internet access, you can place your cluster behind a proxy, or you can perform a _restricted network installation_. In a restricted network installation, you can download the images that are required to install a cluster, place them in a mirror registry, and use that data to install your cluster. While you require internet access to pull images for platform containers, with a restricted network installation on vSphere or bare metal infrastructure, your cluster machines do not require direct internet access.
```

Comment:
Is GCP UPI restricted network supported for 4.2? I don't see the docs in the GCP UPI PR. If not, then I'd be careful about generally saying that for installs on UPI that you can do a restricted network install.